content/docs/command-reference/add.md

```diff
@@ -6,11 +6,11 @@ Track data files or directories with DVC, by creating a corresponding
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f <filename>]
-               targets [targets ...]
+usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
+               [-f <filename>] targets [targets ...]
 
 positional arguments:
-  targets               Input files/directories to add.
+  targets               Files or directories to add
```

Comment:
Needs core repo change.

Reply:
Mah we'll probably need to check all the commands to match usage and options from docs to core after this whole 1.0 drive... Created iterative/dvc/issues/4024